src/main/java/de/unistuttgart/iste/gits/flashcardservice/validation/FlashcardValidator.java

```diff
@@ -0,0 +1,21 @@
+package de.unistuttgart.iste.gits.flashcardservice.validation;
+import org.springframework.stereotype.Component;
+
+@Component
+
+public class FlashcardValidator {
+
+    public void validateCreateFlashcardInput(CreateFlashcardInput flashcardInput) {
+        // TODO implement validation
```

Comment:
Is there nothing to validate?

Reply:
I have to do this, think about this and discuss with you